compiler/src/dotty/tools/dotc/config/Printers.scala

```diff
@@ -17,6 +17,7 @@ object Printers {
   val checks: Printer = noPrinter
   val config: Printer = noPrinter
   val cyclicErrors: Printer = noPrinter
+  val debug = noPrinter
```

Comment:
I think you missed the type annotation here.
```suggestion
  val debug: Printer = noPrinter
```